solr/core/src/java/org/apache/solr/response/transform/ChildDocTransformer.java

```diff
@@ -131,6 +131,12 @@ public void transform(SolrDocument rootDoc, int rootDocId) {
 
           // load the doc
           SolrDocument doc = searcher.getDocFetcher().solrDoc(docId, childReturnFields);
```

Comment:
Perhaps this should be add to DocFetcher#solrDoc in the future?
This is just a thought.

Reply:
I was thinking the same thing!  But yeah; out of scope for this PR.  CC @ErickErickson